### DISCUSSION TITLE
How to  use multi Configuration, multi target project?

### DISCUSSION THREAD

**Discussion by seubseub** [swiftyfinch/Rugby#110] [Q&A] [answered]
Thank you for your great project 👍 

I want to apply Rugby for team project.

But My Project have 

Multi Target : 'A', 'B'
Multi Configuration: 'Debug Beta', 'Debug Real'.. etc.

I tried default rugby but build failed.

fail reason is same as https://github.com/swiftyfinch/Rugby/discussions/109.

So I use custom plans file.
1 Exclude xcframeworks.
2 set custom plan
<img width="600" alt="image" src="https://user-images.githubusercontent.com/16539714/147132685-762416d1-f5cb-4951-8ca4-d2fc7fdc8c53.png">

And this is result
![image](https://user-images.githubusercontent.com/16539714/147132494-6876651b-b522-448c-ba5d-679fee0db598.png)

But When build Xcode,
I can't find include pod ex) DeviceKit.

Could you provide multi cofiguration, multi target example?

Could you help this problem?

Thank you.

**Comment by swiftyfinch** [comment, marked as answer]
@seubseub Hey!

1) If you want to build several configs, you can read this discussion https://github.com/swiftyfinch/Rugby/discussions/88#discussioncomment-1687810
And also, subscribe to this feature https://github.com/swiftyfinch/Rugby/issues/93

2) I didn't see any specific error in your screens. You can try to use `rugby doctor` and send more info here.
Or prepare a project in which I can reproduce your error. And then I try to understand and help you.
I've already written something similar in #109

3) You can start with simpler command. Like: `rugby cache --config Debug Beta`.
And if it's OK, you can try to prepare more complex `.rugby/plans`

4) Rugby 🏈 excludes XCFrameworks, so I'm not sure why you still need to do it manually.
Maybe there is some problem.

**Reply by seubseub** [reply]
@swiftyfinch 
1 -> https://github.com/swiftyfinch/Rugby/discussions/88#discussioncomment-1687810 I already read this discussion.
I think team project is some problem :(

2. OK. if I have something to share, then share the problem.
3, 4 : Thanks : )

**Reply by seubseub** [reply]
Then how about widget target??

**Reply by swiftyfinch** [reply]
I'm not sure that understand about `widget target`. Is it related to https://github.com/swiftyfinch/Rugby/issues/93#issuecomment-999271761?
I think if someone needs to different sdk just open a new issue. Afterward, we collect them and add to `sdk` enum.

**Reply by seubseub** [reply]
@swiftyfinch 
this widget target.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/16539714/147209304-5ff16e50-bea2-4df4-8350-3c8e04a9fc83.png">
